scripts/install_ffmpeg_cuda.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Step 1: Install CUDA Toolkit
+# Add the NVIDIA repository for CUDA and install CUDA toolkit version 12.6
+dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel8/x86_64/cuda-rhel8.repo
+dnf clean all
+dnf -y install cuda-toolkit-12-6
+
+# Set CUDA environment variables for PATH and LD_LIBRARY_PATH
+export PATH=/usr/local/cuda/bin:${PATH}
+export LD_LIBRARY_PATH=/usr/local/cuda/lib64
+
+# Verify nvcc installation (CUDA compiler)
+nvcc --version
+
+# Step 2: Install dependencies required for FFmpeg
+# Install libraries and tools required for FFmpeg compilation
+dnf install -y libtool \
+    glibc \
+    glibc-devel \
+    numactl \
+    numactl-devel \
+    openssl \
+    yasm \
+    pkg-config \
+    openssl-devel
+
+# Step 3: Install x264
+# Add the RPM Fusion repository and install x264 and its development files
+dnf install -y https://download1.rpmfusion.org/free/el/rpmfusion-free-release-8.noarch.rpm
+dnf install -y x264 x264-devel
+
+# Step 4: Install NVENC codec headers
+# Install the necessary development tools and clone the nv-codec-headers repository
+dnf groupinstall -y "Development Tools"
```

Comment:
what dev tools do we need here. This group install will install hundreds of dependencies we might not need in our final image. which includes Java 8.